### PR TITLE
refactor(editor): Refactor utils files and mixins

### PR DESCRIPTION
**Pull request by MiloradFilipovic**
Also fixes N8N-5474

### CODE REVIEW

File: packages/editor-ui/src/components/Node/NodeCreation.vue [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 <script lang="ts">
 import Vue from "vue";
-import * as CanvasHelpers from "@/views/canvasHelpers";
+import { getMidCanvasPosition } from '@/utils';
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/Node/NodeCreation.vue:21]: Nice.

**Review by alexgrozav — DISMISSED** [review]
Awesome work, Milorad!

**Review by alexgrozav — APPROVED** [review]
🎉

**Comment by janober** [comment]
Got released with `n8n@0.204.0`

**Pull request merged**